packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1000,6 +1000,7 @@ oie.okta_verify.totp.enterCodeText = Enter code from Okta Verify app
 oie.okta_verify.push.title = Get a push notification
 oie.okta_verify.push.sent = Push notification sent
 oie.okta_verify.push.resend = Resend push notification
+oie.okta_verify.sendPushButton = Send push
```

Comment:
Calling this out as an FYI, but this breaks existing customizations for customers who have altered the existing [`oktaverify.send`](https://github.com/okta/okta-signin-widget/blob/c7aaf9159fdf6af7a2102d485016d7817e50f670/packages/%40okta/i18n/src/properties/login.properties#L570) key.

I'd suggest confirming with product (and reach out to DevEx who maintains the Changelog for this repo) that customers migrating will need to update their customization keys.